### PR TITLE
mailchimp

### PR DESCRIPTION
**Pull request by Aranzazu0611**

### CODE REVIEW

File: staff/aranzazu-perdomo/mailchimp/index.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        <ul class="footer-list">
+				<h3>Contact Us</h3>
+				<li><a href="#">Contact MailChimp</a></li>
+				<li><a href="#">Partner With Us</a></li>
+				<li><a href="#">Abuse Desk</a></li>
+				<li><a href="#">MailChimp Status</a></li>
+		</ul>
+        
+        <ul class="footer-list">
+				<h3>Legal Info</h3>
```
Comment (mikelpmc) [inline staff/aranzazu-perdomo/mailchimp/index.html]: Only li should be inside an ul

File: staff/aranzazu-perdomo/cv/index.html [hunk 2]
```diff
@@ -5,20 +5,21 @@
...
     <header>
      <h1> Curriculum</h1>
      
+     
     </header>
     <main>
     
-    <section>
-    
+    <section class="datos">
```
Comment (mikelpmc) [inline staff/aranzazu-perdomo/cv/index.html]: try to use English if possible.

File: staff/aranzazu-perdomo/cv/index.html [hunk 3]
```diff
@@ -5,20 +5,21 @@
...
+        <img src="john-doe.jpg" alt="foto-jhon">
         <h1><strong> Datos:</strong></h1>
-        <ul>
+        <ul class="datos-base">
```
Comment (Javier1177) [inline staff/aranzazu-perdomo/cv/index.html]: You should close ul

File: staff/aranzazu-perdomo/mailchimp/index.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                    <img src="./images/home/unfurlr-thumb.jpg" alt="">
+                    <p  class="card-subtitle">App Spotlight:</p>
+                    <h4 class="card-title">Unfurlr</h4>
+                    <p  class="card-description">Unfurlr shows you the destination of shortened URLs befoure you click, so you can avoid phishing attacks and other nerfarious links</p>
+                    <button class="card-btn">Learn More<i class="icon-play"></i></button>
+            
+            </article>
+
+            <article class="card">
+                    <img src="./images/home/golden-monkeys.jpg" alt="">
```
Comment (giodelabarrera) [inline staff/aranzazu-perdomo/mailchimp/index.html]: Como buena practica, agregar el titulo de la card en el atributo alt del img
```html
<img src="./images/home/golden-monkeys.jpg" alt="Golden Monkeys">
```

File: staff/aranzazu-perdomo/spotify-app/scripts/logic.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+                    if (request.status === 200) {
+                        var res = JSON.parse(request.responseText);
+
+                        resolve(res);
+                    } else reject(Error('request error, status ' + request.status));
+                }
+            };
+
+            var url = 'https://api.spotify.com/v1' + path;
+
```
Comment (gbaste) [inline staff/aranzazu-perdomo/spotify-app/scripts/logic.js]: Hi! You need to clean the spaces

File: staff/aranzazu-perdomo/spotify-app/scripts/logic.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
+var logic = {
```
Comment (giodelabarrera) [inline staff/aranzazu-perdomo/spotify-app/scripts/logic.js]: missing
```js
'use strict';
```

File: staff/aranzazu-perdomo/spotify-app/scripts/main.js [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+            detailContainer.clear();
+        })
+        .catch(function (error) {
+            alert('Sorry, we have temporary problem, try again later.');
+        });
+});
+
+var resultsAlbum = new ResultsList();
+var resultsTracks=new ResultsList();
```
Comment (giodelabarrera) [inline staff/aranzazu-perdomo/spotify-app/scripts/main.js]: You have forgotten add spaces in this line.

File: staff/aranzazu-perdomo/cv/index.html (hunk 2 above)
Comment (Aranzazu0611) [reply staff/aranzazu-perdomo/cv/index.html]: ok! i changed.

File: staff/aranzazu-perdomo/cv/index.html (hunk 3 above)
Comment (Aranzazu0611) [reply staff/aranzazu-perdomo/cv/index.html]: done

File: staff/aranzazu-perdomo/mailchimp/index.html (hunk 4 above)
Comment (Aranzazu0611) [reply staff/aranzazu-perdomo/mailchimp/index.html]: done!

File: staff/aranzazu-perdomo/mailchimp/index.html (hunk 1 above)
Comment (Aranzazu0611) [reply staff/aranzazu-perdomo/mailchimp/index.html]: ok!

File: staff/aranzazu-perdomo/spotify-app/scripts/logic.js (hunk 5 above)
Comment (Aranzazu0611) [reply staff/aranzazu-perdomo/spotify-app/scripts/logic.js]: Done!

File: staff/aranzazu-perdomo/spotify-app/scripts/logic.js (hunk 6 above)
Comment (Aranzazu0611) [reply staff/aranzazu-perdomo/spotify-app/scripts/logic.js]: ok!

File: staff/aranzazu-perdomo/spotify-app/scripts/main.js (hunk 7 above)
Comment (Aranzazu0611) [reply staff/aranzazu-perdomo/spotify-app/scripts/main.js]: ok!

**Pull request merged**